app/client/components/management/exportManager.jsx

```diff
@@ -94,6 +98,18 @@ class ExportManager extends React.Component {
   }
 
   render() {
+    let transferToggle = (
```

Comment:
shifted from below to a var so we can optionally render it for spreadsheet export only